src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt

```diff
@@ -212,6 +212,7 @@ fun GetDelegateTokenRequest.validate(): GetDelegateTokenRequest {
 
 // Validations used by ValidationBuilders
 
+const val NAME_REGEX = "^[a-zA-Z0-9_\\s]*\$"
```

Comment:
I think we must expand the regex here for French or German names? May be we can do it when someone asks.

Reply:
Will do it once the ask comes